packages/transformer/src/IModelExporter.ts

```diff
@@ -264,12 +277,11 @@ export class IModelExporter {
    * you pass to [[IModelExporter.exportChanges]]
    */
   public async initialize(options: ExporterInitOptions): Promise<void> {
-    const hasChangeData = options.startChangeset || options.changesetRanges || options.changedInstanceIds;
+    const hasChangeData = "csFileProps" in options || "startChangeset" in options || "changesetRanges" in options || "changedInstanceIds" in options;
     if (this._sourceDbChanges || !this.sourceDb.isBriefcaseDb() || !hasChangeData)
       return;
 
-    this._sourceDbChanges = options.changedInstanceIds
-      ?? await ChangedInstanceIds.initialize({ iModel: this.sourceDb, ...options });
+    this._sourceDbChanges = "changedInstanceIds" in options ? options.changedInstanceIds : await ChangedInstanceIds.initialize({ iModel: this.sourceDb, ...options });
```

Comment:
rather than returning undefined from `initialize` (which perhaps the name should be deprecated and it be renamed to `create`), we should return an empty ChangedInstanceIds. Either the caller should resolve if there were 0 changesets, or we should throw an error on 0 changesets and let the caller catch that... I think. Can discuss when you're back